miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -509,7 +543,6 @@ export.get_procedure_info
   dup push.255 neq
   # => [is_255, index]
 
-  # load procedure information from memory
   if.true
```

Comment:
I'd love to be able to get rid of this condition. If we cannot do it in this PR, let's do it immediately after.

Reply:
Opened issue here: https://github.com/0xPolygonMiden/miden-base/issues/851